### PR TITLE
Working YCSB workload for couchbase infra

### PR DESCRIPTION
**Pull request by dustinblack**
YCSB base workload now functional for couchbase infra and ready for use with $other infra in the future.

This commit was rebased on the node for my couchbase commit in PR #8 so it includes commits also in that PR. I'm not entirely sure how git handles resolving that, so if I need to delete and re-submit this PR after #8 is merged to master, let me know.

### CODE REVIEW

**Comment by aakarshg** [comment]
If this has dependency on #8 then I'd suggest that we wait until #8 is merged and lands.

File: deploy/crds/bench_v1alpha1_bench_cr.yaml [hunk 1]
```diff
@@ -20,3 +31,10 @@ spec:
     runtime: 57
     rw: write
     filesize: 1
+  ycsb:
```
Comment (jtaleric) [inline deploy/crds/bench_v1alpha1_bench_cr.yaml]: Did you mean to include this?

File: playbook.yml [hunk 2]
```diff
@@ -4,6 +4,12 @@
   - include_role:
       name: "uperf-bench"
     when: uperf is defined and uperf.pair > 0
+  - include_role:
+      name: "couchbase-infra"
+    when: couchbase is defined and couchbase.servers.size > 0
   - include_role:
       name: "fio-bench"
     when: fio is defined and fio.clients > 0
+  - include_role:
```
Comment (jtaleric) [inline playbook.yml]: I think the streams are crossed...

**Comment by dustinblack** [comment]
I feel like this is just going to goof up git commit history as it is. I will wait for PR #8 to merge, then I'll make sure this one is rebased and will submit a new PR.

**Pull request closed without merging**